### PR TITLE
chore: Update aries-framework-go dependency

### PR DESCRIPTION
**Pull request by DRK3**
Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

**Review by fqutishat — APPROVED** [review]

File: component/storage/couchdb/store.go [hunk 1]
```diff
@@ -333,16 +333,6 @@ func (c *StoreCouchDB) Iterator(startKey, endKey string) storage.StoreIterator {
...
-	resultRows, err := c.db.Find(context.Background(), findQuery)
-	if err != nil {
-		return nil, fmt.Errorf("failed to query CouchDB using the find endpoint: %w", err)
-	}
-
-	return &couchDBResultsIterator{log: c.log, store: c, resultRows: resultRows}, nil
-}
-
 type couchDBResultsIterator struct {
 	log        logger
```
Comment (soluchok) [inline component/storage/couchdb/store.go:337]: Remove logger as well.

Comment (soluchok) [reply component/storage/couchdb/store.go:337]: This one also needs to be removed https://github.com/hyperledger/aries-framework-go-ext/blob/main/component/storage/couchdb/store.go#L363

**Review by rolsonquadras — APPROVED** [review]

File: component/storage/couchdb/store.go (hunk 1 above)
Comment (DRK3) [reply component/storage/couchdb/store.go]: Good catch - done. I also removed the WithLoggerOption function.

**Review by soluchok — COMMENTED** [review]
@DRK3 you committed empty file `test/component/storage/.golangci.yml`

**Comment by DRK3** [comment]
> @DRK3 you committed empty file `test/component/storage/.golangci.yml`

@soluchok fixed.

**Review by soluchok — APPROVED** [review]

**Review by fqutishat — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

**Review by llorllale — APPROVED** [review]

**Review by troyronda — APPROVED** [review]

**Pull request merged**